<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillinBlankQ</class>
 <widget class="QWidget" name="FillinBlankQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>正在答题</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QTextEdit" name="question_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>700</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ansArea_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ansArea_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作答区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>⇨</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>⇦</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answer_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>查看答案</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>（如有多个答案，请用中文顿号“、”隔开）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="index_lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 193, 179);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
